cypress.base.config.ts

```diff
@@ -23,7 +22,7 @@ export const baseConfig: Cypress.ConfigOptions = {
     setupNodeEvents(on, config) {
       // implement node event listeners here
       // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-var-requires
-      codeCoverage(on, config);
+      // codeCoverage(on, config);
```

Comment:
Lets remove this code all together rather than commenting out.

Reply:
done